site/_apps/zuix-hn.txt

```diff
@@ -6,19 +6,19 @@ library: 'ZUIX'
 libraries: 
     - name: 'ZUIX'
 module-bundling: 'zuix-bundler'
-service-worker: 'Application Shell + Workbox'
+service-worker: 'Application Shell + Workbox data caching'
 perfomance-patterns: 'Lazy loaded components with caching'
 server-side-rendering: 'None'
 api: 'Hacker News Firebase API'
 hosting: 'Firebase'
 authors: 
     - name: 'Gene'
 lighthouse: '91/100'
-interactive-em: '5.1s'
-interactive-faster-3g: '4.3s'
-lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170613_Y4_3f81ed03de518aaecc6a8823e74ec401&run=3
-wpt-em-link: https://www.webpagetest.org/result/170613_22_be140a755468b49f0e8480eeb79149ed/
-wpt-faster-3g-link: https://www.webpagetest.org/result/170613_Y4_3f81ed03de518aaecc6a8823e74ec401/
+interactive-em: '4.8s'
```

Comment:
Thanks @addyosmani for reviewing. I updated the numbers but also re-run the Faster-3G test since it looked strange that EM had a lower interactive-time. 
https://www.webpagetest.org/result/170715_PW_749f08a967caa216ca8cd749346601cb/